source/components/molecules/ToastNotification/ToastNotifications.tsx

```diff
@@ -1,19 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Notification } from '../../../store/NotificationContext';
+import { Notification, Popup } from '../../../store/NotificationContext';
 import Toast from './Toast';
+import PopupComponent from '../../../containers/Popup/Popup';
```

Comment:
This is not a good practice at all ie making a molecule dependent on a container component. Molecules should only be dependent on atom components. 

Also notice that there is other dependencies that are making this component dependent on context etc. In other terms this does not feel like a molecule component at all. It has way to much logic in it.

Reply:
Yeah, this thing is probably not a molecule component. More like a container, perhaps.